src/microbe_stage/Patch.cs

```diff
@@ -88,6 +88,42 @@ public Patch(LocalizedString name, int id, Biome biomeTemplate, PatchSnapshot cu
     [JsonProperty]
     public int[] Depth { get; private set; } = { -1, -1 };
 
+    public bool Explored
+    {
+        get => VisibilityState == MapElementVisibility.Explored;
+        set
+        {
+            if (value)
+            {
+                VisibilityState = MapElementVisibility.Explored;
+                Region.VisibilityState = MapElementVisibility.Explored;
+            }
+            else
+            {
+                VisibilityState = MapElementVisibility.Undiscovered;
+                Region.VisibilityState = MapElementVisibility.Undiscovered;
+            }
+
+            if (value)
```

Comment:
You already check for `if (value)` a few lines above this so I think this if's content should go there.